CONTRIBUTING.md

```diff
@@ -21,8 +21,8 @@ After submitting a PR which meets all criteria from the previous section, it wil
 
 Currently, the list of people maintaining the fork include:
```

Comment:
Hmm, our driver is not a fork, or is it? I think it's a leftover from the very beginning, when it rose as a fork of `cdrs`?
Anyway, this should probably be changed: `the fork` -> `the driver`.

Reply:
> Hmm, our driver is not a fork, or is it? I think it's a leftover from the ancient time of the driver, when it rose as a fork of `cdrs`?

It was never a fork of `cdrs`. Perhaps here "the fork" meant "the main fork", I'm not sure.